tests/posix/common/src/timer.c

```diff
@@ -104,7 +104,7 @@ ZTEST(timer, test_CLOCK_MONOTONIC__SIGEV_THREAD)
 
 ZTEST(timer, test_timer_overrun)
 {
-	struct sigevent sig = { 0 };
+	struct sigevent sig = {0};
```

Comment:
My preference would be to keep changes that are exclusively formatting in a separate commit, if they are necessary.

Reply:
commit restored (not squashed)

Reply:
This change (the 2nd commit) should have a commit message

Reply:
This change is still not strictly unnecessary. If you feel we need it, then please add it as a 3rd commit, or simply undo this change (you night need to disable auto-formatting).